docs/installation/fedora.md

```diff
@@ -5,108 +5,47 @@ description = "Instructions for installing Docker on Fedora."
 keywords = ["Docker, Docker documentation, Fedora, requirements,  linux"]
 [menu.main]
 parent = "smn_linux"
+weight=-3
 +++
 <![end-metadata]-->
 
 # Fedora
 
-Docker is supported on the following versions of Fedora:
-
-- Fedora 21
-- Fedora 22 
-
-This page instructs you to install using Docker-managed release packages and
-installation mechanisms. Using these packages ensures you get the latest release
-of Docker. If you wish to install using Fedora-managed packages, consult your
-Fedora release documentation for information on Fedora's Docker support.
+Docker is supported Fedora version 20 and 21. This page instructs you to install
+using Docker-managed release packages and installation mechanisms. Using these
+packages ensures you get the latest release of Docker. If you wish to install
+using Fedora-managed packages, consult your Fedora release documentation for
+information on Fedora's Docker support.
 
 ## Prerequisites
 
 Docker requires a 64-bit installation regardless of your Fedora version. Also, your kernel must be 3.10 at minimum. To check your current kernel
 version, open a terminal and use `uname -r` to display your kernel version:
 
-    $ uname -r 
-    3.19.5-100.fc20.x86_64
+    $ uname -r
+    3.19.5-100.fc21.x86_64
 
 If your kernel is at a older version, you must update it.
 
 Finally, is it recommended that you fully update your system. Please keep in
 mind that your system should be fully patched to fix any potential kernel bugs. Any
-reported kernel bugs may have already been fixed on the latest kernel packages 
+reported kernel bugs may have already been fixed on the latest kernel packages
 
 
 ## Install
 
-There are two ways to install Docker Engine.  You can use `curl` with the  `get.docker.com` site. This method runs an installation script which installs via the `yum` package manager. Or you can install with the `yum` package manager directly yourself.
-
-### Install with the script
+There are two ways to install Docker Engine.  You can install with the `yum` package manager directly yourself. Or you can use `curl` with the  `get.docker.com` site. This second method runs an installation script which installs via the `yum` package manager.
```

Comment:
Perhaps describe the second method as ".... installation script which performs these steps automatically."